package.json

```
@@ -7,6 +7,7 @@
     "build": "next build",
     "start": "next start",
     "lint": "eslint .",
+    "stylelint": "stylelint '**/*.css'",
```

You can`t run i think...

```suggestion
    "stylelint": "stylelint \"**/*.css\"",
```

this can help

---

Thanks for the suggestion, but it's working with the `'`'s :P